app/controllers/blogs_controller.rb

```diff
@@ -1,6 +1,7 @@
 class BlogsController < ApplicationController
   # %i[ show edit update destroy ] = [:show, :edit, :update, :destroy]
   before_action :set_blog, only: %i[show edit update destroy]
+  before_action :authenticate_user!
```

Comment:
```suggestion
  before_action :authenticate_user!, only: %i[ show edit create update destroy ]
```
indexとshowに関しては、含まれない気がします。

Reply:
こちら修正しました。
ブログ作成時に、コントローラーまで渡らないようになりましたが、blog内のエラーではなく、diveseの認証エラーに入ってるのでそれを表示させたいので方法を教えてほしいです。